cmd/token.go

```diff
@@ -80,7 +82,7 @@ func loginCmdRun(cmd *cobra.Command, args []string) error {
 		p.URL = login.ValidateTokenURL
 		r, err := login.ValidateCredentials(p)
 		if err != nil {
-			return fmt.Errorf("Failed to validate: %v", err.Error())
+			return fmt.Errorf("failed to validate: %v", err.Error())
```

Comment:
@Xemdo Did you still want to keep this uppercase or lowercase?

Reply:
Lowercase is fine, since it matches Twitch's errors returned over HTTP. However since this is on console, it doesn't really matter too much as long as its readable.